pom.xml

```diff
@@ -160,13 +160,13 @@
     <commons-codec.version>1.15</commons-codec.version>
     <eddsa.version>0.3.0</eddsa.version>
     <ethereum-core.version>1.12.0-v0.5.0</ethereum-core.version>
-    <grpc.version>1.33.0</grpc.version>
+    <grpc.version>1.39.0</grpc.version>
     <jackson-databind.version>2.9.10.7</jackson-databind.version>
     <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
-    <hapi-proto.version>0.16.0-rc.1</hapi-proto.version>
+    <hapi-proto.version>0.17.0-SNAPSHOT</hapi-proto.version>
     <javax-inject.version>1</javax-inject.version>
     <log4j.version>2.13.2</log4j.version>
-    <netty.version>4.1.51.Final</netty.version>
+    <netty.version>4.1.65.Final</netty.version>
     <swirlds.version>0.16.0</swirlds.version>
```

Comment:
Is there any reason not to use 4.1.66.Final?

Reply:
netty `4.1.66.Final` is fine with me.

Reply:
Changed to `4.1.66.Final`.